### ISSUE TITLE
Replace unordered lists with a table

### ISSUE THREAD

**Issue by BatemanVO** [aggiewb/salmon-cookies#6]
To be completed after #5.

Pat sent us a new document [which can be found here](https://github.com/codefellows/amazon-201d1/tree/master/class-07-object-constructors-html-tables/lab). Although the original document says to style the table however you want, we'd like the table to match the example closely:
- Table header has bold text
- Within the table body, every even row has a different background color than every odd row
- Every store location is a row in the table, with each hour of business being a column
- The table should have a totals row, totaling all stores' cookies for that hour
- The last column of the table should be a totals column, totaling all stores' cookies for that day
- The last cell in the table should be a totals cell for all stores for that day

The only HTML hardcoded in the `sales.html` file for the table should be:
- `<table>`
- `<caption>`
- `<thead>`
- `<tbody>`
- `<tfoot>`

All other information on the table should be dynamically generated and appended to the DOM via JavaScript. The `<thead>` should contain all business hours, the `<tbody>` should contain each store's data, and the `<tfoot>` should contain the totals row.

The header and footer row should have their own separate functions that append them to the table.
 
Additionally, a new method should be added to the constructor prototype created for #5. This method should be called `render` and should append a given store's location's table row to the table.

The table [must meet the WebAIM accessibility standards](https://webaim.org/techniques/tables/data) (try using the inspector to inspect some of their example tables).

**Comment by BatemanVO** [comment]
@aggiewb Some QA/UAT issues have been created and assigned to you. I've assigned the ticket back to you, but leave it in QA/UAT. 

Small changes for QA/UAT do not require formal code review. Up to dev discretion on what a small change is, but changes must be in the `staging` branch before flipping the assignee back to QA/UAT resources (me).

Either open new PRs to `staging`, then merge them to both `staging` and your `feature` branch, or push new commits directly to your `feature` branch and then do another PR from your `feature` branch to `staging`. Don't push directly to `staging`.

If you have questions, let me know.

**Comment by BatemanVO** [comment]
QA and UAT complete, moving to Ready For Deployment.
